src/panels/config/integrations/integration-panels/zwave_js/zwave_js-logs.ts

```diff
@@ -118,10 +118,17 @@ class ZWaveJSLogs extends SubscribeMixin(LitElement) {
     if (ev.target === undefined || this._logConfig === undefined) {
       return;
     }
-    if (this._logConfig.level === ev.target.selected) {
+    const selected = ev.target.selected;
+    if (this._logConfig.level === selected) {
       return;
     }
-    setZWaveJSLogLevel(this.hass!, this.configEntryId, ev.target.selected);
+    setZWaveJSLogLevel(this.hass!, this.configEntryId, selected);
+    this._logConfig.level = selected;
+    this._textarea!.value += `${this.hass.localize(
+      "ui.panel.config.zwave_js.logs.log_level_changed",
+      "level",
+      selected.charAt(0).toUpperCase() + selected.slice(1)
+    )}\n`;
```

Comment:
```suggestion
    this._textarea!.value += `${this.hass.localize(
      "ui.panel.config.zwave_js.logs.log_level_changed",
      {level:
      selected.charAt(0).toUpperCase() + selected.slice(1)}
    )}\n`;
```

Reply:
We could also localize the log level...

Or we could create a lookup object, that could also be used during the rendering of the dropdown... but this is also fine...

Reply:
I have no opinion, just chose the solution I was able to find 🙂